generators/client-2/templates/src/main/webapp/_index.html

```diff
@@ -84,20 +84,24 @@
         r.parentNode.insertBefore(e,r)}(window,document,'script','ga'));
         ga('create','UA-XXXXX-X');ga('send','pageview');
     </script>-->
-    <script src="vendor/jquery/dist/jquery.js"></script>
+    <!--<script src="vendor/jquery/dist/jquery.js"></script>-->
     <!-- build:js app/vendor.js -->
     <!-- bower:js -->
     <!-- endinject -->
     <!-- endbuild -->
 
-    <script src="vendor/core-js/client/shim.min.js"></script>
+    <!--<script src="vendor/core-js/client/shim.min.js"></script>
     <script src="vendor/zone.js/dist/zone.js"></script>
     <script src="vendor/reflect-metadata/Reflect.js"></script>
     <script src="vendor/systemjs/dist/system.js"></script>
     <script src="system.config.js"></script>
     <script>
         System.import('app').catch(function(err){ console.error(err); });
-    </script>
+    </script>-->
```

Comment:
Have you thought about leveraging html-webpack-plugin. When you start to use deterministic hashing in your bundles, this will get complex to handle adding scripts to your html. The aformentioned plugin can handle this (and a bunch of other awesome features as well).